docs/source/optim.rst

```diff
@@ -31,6 +31,17 @@ a ``params`` key, containing a list of parameters belonging to it. Other keys
 should match the keyword arguments accepted by the optimizers, and will be used
 as optimization options for this group.
 
+For example, this is very useful when one wants to specify per-layer learning rates::
+
+    optim.SGD([
+                    {'params': model.base.parameters()},
+                    {'params': model.classifier.parameters(), 'lr': 1e-3}
+                ], lr=1e-2, momentum=0.9)
+
+This means that ``model.base``'s parameters will use the default learning rate of ``1e-2``,
+``model.classifier``'s parameters will use a learning rate of ``1e-3``, and a momentum of
```

Comment:
```suggestion
    optim.SGD([
                    {'params': model.base.parameters(), 'lr': 1e-2},
                    {'params': model.classifier.parameters()}
                ], lr=1e-3, momentum=0.9)

This means that ``model.base``'s parameters will use a learning rate of ``1e-3``,
``model.classifier``'s parameters will use the default learning rate of ``1e-3``, and a momentum of
```

Might need some spacing updates, but let's also change the phrasing here to be more consistent with expectation. Recently, we determined SGD's default LR to be 1e-3, so making that the "default" in this case will raise fewer eyebrows.